tutor/src/screens/my-courses/dashboard/use-displayed-offerings.tsx

```diff
@@ -1,17 +1,17 @@
 import React, { useState, useEffect, useMemo } from 'react'
-import { filter, includes, indexOf, findIndex } from 'lodash'
+import { filter, includes, indexOf, findIndex, map } from 'lodash'
 import { ID, Offering } from '../../../store/types'
 import UiSettings from 'shared/model/ui-settings'
-import { useDisplayedCourseOfferingIds, useAllCourses } from '../../../store/courses'
-import { useAvailableOfferings } from '../../../store/offering'
+import { useAllCourses } from '../../../store/courses'
+import { useAvailableOfferings, useDisplayedOfferingsIds } from '../../../store/offering'
 
 type ReturnUseDisplayedOfferings = [ID[], React.Dispatch<React.SetStateAction<ID[]>>, Offering[], (offeringId: ID, flow?: string) => void]
 
 const useDisplayedOfferings = () : ReturnUseDisplayedOfferings => {
     // getting all the data: offerings and courses
     const courses = useAllCourses()
     const offerings = useAvailableOfferings(courses)
-    const [displayedOfferingIds, setDisplayedOfferingIds] = useState<ID[]>(useDisplayedCourseOfferingIds())
+    const [displayedOfferingIds, setDisplayedOfferingIds] = useState<ID[]>(map(useDisplayedOfferingsIds(courses)))
```

Comment:
what's the purpose of the `map` here?

Reply:
nothing, I think I forgot to delete it when I was moving logic around.